logicmonitor/sensors/logicmonitor_sensor.py

```diff
@@ -0,0 +1,102 @@
+import json
+import requests
+import sensor_constants as C
+from flask import request, Flask
+from st2reactor.sensor.base import Sensor
+
+
+class LogicMonitorSensor(Sensor):
+    def __init__(self, sensor_service, config=None):
+        self._config = config
+        self._sensor_service = sensor_service
+
+        self._host = "0.0.0.0"
+        self._port = "5000"
+
+        if config != None:
+            self._auth_enabled = config.get("auth_enabled")
+
+        self._app = Flask(__name__)
+        self._log = self._sensor_service.get_logger(__name__)
+
+    def setup(self):
+        pass
+
+    def run(self):
+        @self._app.route("/", methods=["POST"])
+        def handle_webhook():
+
+            # Get JSON data from POST request
+            try:
+                data = json.loads(request.data)
+                self._log.info(C.LOG_LOADING_DATA)
+            except Exception:
+                self._log.info(C.LOG_FAILED_TO_PARSE_JSON)
+                return C.RES_FAILED_TO_PARSE_JSON
+
+            # Get API Key from POST request
+            try:
+                _apiKey = data[C.API_KEY_KEY]
+                self._log.info(C.LOG_API_KEY_EXISTS)
+            except Exception:
```

Comment:
This is a very broad except block which is generally a bad idea. I think you only care about KeyError here, right?
```suggestion
            except KeyError:
```

Reply:
This is a good point but don't want to push untested changes even if they are 99.99% likely to succeed. Right now, just trying to put the pack into Exchange.